frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -244,8 +246,8 @@ private void serveCheckComponentName(ResourceRequest request, ResourceResponse r
             // tokens:
             // \W = a non word character, so not in [a-zA-Z_0-9]
             Set<String> splitCName = Sets.newHashSet(cName.split("\\W"));
-            // to find tomcat even on a cName of tomca, add * to the search term
-            Set<String> splitExtendedCName = splitCName.stream().map(v -> v + "*").collect(Collectors.toSet());
+            // to find tomcat even on a cName of tomcat, add * to the search term
```

Comment:
This wasn't a typo. The comment doesn't make sense now.

Reply:
yes.... :+1: